helmfile/values/gatekeeper/constraints.yaml.gotmpl

```diff
@@ -10,6 +10,9 @@ requireResourceRequests:
 disallowedTags:
     enable: {{ .Values.opa.disallowedTags.enabled }}
     enforcementAction: {{ .Values.opa.disallowedTags.enforcement }}
+rejectLoadBalancerService:
+    enable: {{ .Values.opa.rejectLoadBalancerService.enabled }}
```

Comment:
As we discussed earlier it is a bit confusing that we are using "enabed" in the config and "enable" in the chart.  If you have time then feel free to refactor that so that all these policies are using "enabled" everywhere. But this is outside the scope of your task, so feel free to skip this if you want.
If you fix this, then keep it as a separate commit.